### ISSUE TITLE
GCHP 14.3.0-rc.0 runs crashing when some diagnostic collections are turned on

### ISSUE THREAD

**Issue by msulprizio** [geoschem/geos-chem#2135]
### Name and Institution (Required)

Name: Melissa Sulprizio
Institution: Harvard / GCST

### Description of your issue or question

When turning on additional diagnostic collections (`CloudConvFlux, ConcAfterChm, DryDep, LevelEdgeDiags, ProdLoss, StateChm, WetLossConv, WetLossLS`) needed for the GCHP 1-year benchmark run for 14.3.0-rc.0, GCHP crashes with the error:

```
GEOS-Chem ERROR [0041]: Registry pointer not found for DryDepVel_SOAS. Check that the tag (e.g. species) is valid for this diagnostic.
 --> LOCATION: HistoryExports_SetDataPointers
 
GEOS-Chem ERROR [0033]: Registry pointer not found for DryDepVel_SOAS. Check that the tag (e.g. species) is valid for this diagnostic.
 --> LOCATION: HistoryExports_SetDataPointers
 
GEOS-Chem ERROR [0086]: Registry pointer not found for DryDepVel_SO4s. Check that the tag (e.g. species) is valid for this diagnostic.
 --> LOCATION: HistoryExports_SetDataPointers
 
GEOS-Chem ERROR [0007]: Registry pointer not found for DryDepVel_SOAS. Check that the tag (e.g. species) is valid for this diagnostic.
 --> LOCATION: HistoryExports_SetDataPointers
```

Commenting out the `DryDep` collection gets past that error, but then returns a similar error for the `CloudConvFlux` collection:

```
GEOS-Chem ERROR [0041]: Registry pointer not found for CloudConvFlux_XYLE. Check that the tag (e.g. species) is valid for this diagnostic.
 --> LOCATION: HistoryExports_SetDataPointers
 
GEOS-Chem ERROR [0033]: Registry pointer not found for CloudConvFlux_XYLE. Check that the tag (e.g. species) is valid for this diagnostic.
 --> LOCATION: HistoryExports_SetDataPointers
 
GEOS-Chem ERROR [0086]: Registry pointer not found for CloudConvFlux_XYLE. Check that the tag (e.g. species) is valid for this diagnostic.
 --> LOCATION: HistoryExports_SetDataPointers
 
GEOS-Chem ERROR [0007]: Registry pointer not found for CloudConvFlux_XYLE. Check that the tag (e.g. species) is valid for this diagnostic.
 --> LOCATION: HistoryExports_SetDataPointers
```

The relevant files are attached:

- [HISTORY.rc](https://github.com/geoschem/geos-chem/files/14090175/HISTORY.rc.txt)
- [gchp.20190101_0000z.log](https://github.com/geoschem/geos-chem/files/14090179/gchp.20190101_0000z.log.txt)

**Comment by yantosca** [comment]
Also see #2139 #2142 #2143

**Comment by yantosca** [comment]
I believe this error is caused by invalid species in the GCHP HISTORY.rc file for the DryDep collection.  Restricting the list of species to just what we need for benchmarking should prevent this error from occurring.

**Comment by yantosca** [comment]
We can close out this issue as PR #2289 has been merged into 14.4.0.
